l1-contracts/contracts/state-transition/verifiers/ZKsyncOSDualVerifier.sol

```diff
@@ -79,7 +72,7 @@ contract ZKsyncOSDualVerifier is IVerifier {
             revert EmptyProofLength();
         }
 
-        // The first element of `_proof` determines the verifier type (either FFLONK or PLONK).
+        // The first element of `_proof` determines the verifier type.
         uint256 verifierType = _proof[0] & 255;
         uint32 verifierVersion = uint32(_proof[0] >> 8);
         if (
```

Comment:
With the current state looks like fflonk is not used at all, maybe we can remove it from the code at all then?